platforms/ios/framework/src/TGURLHandler.h

```diff
@@ -0,0 +1,77 @@
+//
+//  TGURLHandler.h
+//  TangramMap
+//
+//  Created by Karim Naaji on 11/23/16.
+//  Updated by Matt Blair on 7/16/18.
+//  Copyright (c) 2017 Mapzen. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "TGExport.h"
+
+/**
+ A URL request completion callback, called when a download request of `TGURLHandler` completes an asynchronous request.
+*/
+typedef void(^TGDownloadCompletionHandler)(NSData * _Nullable data, NSURLResponse * _Nullable response, NSError * _Nullable error);
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ A URL Handler interface for creating cancellable asynchronous URL requests.
+ */
+@protocol TGURLHandler <NSObject>
```

Comment:
We should `TG_EXPORT` this thing.

Reply:
I had to look this up when I was setting up symbol exporting - turns out protocols are just interface declarations, there aren't actually any symbols associated with it that we need to export.

Reply:
Same I also looked this up after I commented. Makes sense.